authors/apps/article/serializers.py

```diff
@@ -3,6 +3,7 @@
 from django.core.validators import MinValueValidator, MaxValueValidator
 from django.db.models import Avg
 from rest_framework import serializers
+import readtime
```

Comment:
@Kibetchirchir  order of these imports should be readtime, django, rest_framework and then imports from local files.